arclightning_frontend/games.html

```diff
@@ -1,67 +1,98 @@
+                { name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH' },
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH' },
 
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH'},
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH'},
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH' },
+                { name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH' },
+                { name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH' }
 
-	]
-            },
-        })
-        var keypress = new Vue({
-                el: '#keypress',
-                mounted: function () {
-                    document.addEventListener("keyup", (e) => {
-                        // redirect to demonstration.html if 'L' key is pressed
-                        if (e.keyCode == 76) { window.location.href = 'demonstration.html' }
-                    })
-                }
+            ]
+        },
+        methods: {
+            qrGen: function (hostname) {
+                this.qrStr = '', this.ranHex = ''
+                for (var i = 0; i < 128; i++) { this.ranHex += parseInt(Math.random() * 16, 10).toString(16) }
```

Comment:
I'd prefer to use https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues here because it uses the OS's random generator rather than JS's pseudorandom generator. Also, we can lower the number of bytes to 32. 64 generates massive qr codes that are a bit difficult to read.

Reply:
Updated. Let me know if that works.